content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -61,73 +63,42 @@ This will be set only if you are using a `cloudprovider`. This setting is handy
-
-Kubernetes will automatically spread the Pods in a replication controller or service across nodes in a single-zone cluster (to reduce the impact of failures). With multiple-zone clusters, this spreading behaviour is extended across zones (to reduce the impact of zone failures). This is achieved via _SelectorSpreadPriority_.
-
-_SelectorSpreadPriority_ is a best effort placement. If the zones in your cluster are heterogeneous (for example: different numbers of nodes, different types of nodes, or different pod resource requirements), this placement might prevent equal spreading of your Pods across zones. If desired, you can use homogenous zones (same number and types of nodes) to reduce the probability of unequal spreading.
-
-The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure that Pods, that claim a given volume, are only placed into the same zone as that volume. Volumes cannot be attached across zones.
-
-The actual values of zone and region don't matter. Nor is the node hierarchy rigidly defined.
-The expectation is that failures of nodes in different zones should be uncorrelated unless the entire region has failed. For example, zones should typically avoid sharing a single network switch. The exact mapping depends on your particular infrastructure - a three rack installation will choose a very different setup to a multi-datacenter configuration.
-
-If `PersistentVolumeLabel` does not support automatic labeling of your PersistentVolumes, you should consider
-adding the labels manually (or adding support for `PersistentVolumeLabel`). With `PersistentVolumeLabel`, the scheduler prevents Pods from mounting volumes in a different zone. If your infrastructure doesn't have this constraint, you don't need to add the zone labels to the volumes at all.
-
-{{< note >}} Starting in v1.17, this label is deprecated in favor of [topology.kubernetes.io/zone](#topologykubernetesiozone). {{< /note >}}
-
-## topology.kubernetes.io/region {#topologykubernetesioregion}
+`topology.kubernetes.io/region=us-east-1`
 
 See [topology.kubernetes.io/zone](#topologykubernetesiozone).
 
 ## topology.kubernetes.io/zone {#topologykubernetesiozone}
 
 Example:
 
-`topology.kubernetes.io/region=us-east-1`
-
 `topology.kubernetes.io/zone=us-east-1c`
 
 Used on: Node, PersistentVolume
 
-On the Node: The `kubelet` populates this with the zone information as defined by the `cloudprovider`.
-This will be set only if you are using a `cloudprovider`. However, you should consider setting this
-on the nodes if it makes sense in your topology.
+On Node: The `kubelet` populates this with the information as provided by the `cloudprovider`.  This will be set only if you are using a `cloudprovider`. However, you should consider setting this on nodes if it makes sense in your topology.
+
+On PersistentVolume: The `PersistentVolumeLabel` admission controller automatically adds zone labels to PersistentVolumes, on GCE and AWS.
```

Comment:
I don't think this is true, we actually opted-out of supporting the GA labels via the PVL admission controller. More context in this thread https://github.com/kubernetes/kubernetes/pull/81431#issuecomment-525435518. This may have changed last I checked though. cc @msau42

Reply:
Ah I recall the reasoning now -- it's because volume providers were updated to set `PV.Spec.NodeAffinity` which is the preferred API over labels.

Reply:
So I think the update required for this PR would be something like:
```
On PersistentVolume: topology-aware volume provisioners will automatically set node affinity constraints to PersistentVolumes
```

Reply:
Can you propose better wording?   Or just strike this entirely?

Reply:
The pv label admission controller also sets nodeaffinity but with beta labels too. At the time, we thought this would be fine and we can use csi migration to properly translate the beta labels during scheduling. However I agree this could be confusing and alarming for users that see their PVs provisioned with deprecated labels, and with csi migration timelines slipping, we should probably revisit that decision.

I added some thoughts here: https://github.com/kubernetes/kubernetes/issues/92237#issuecomment-719709773

Reply:
> Do I interpret correctly that the old labels will be respected but the new will not?

Only if the labels are used -- new labels should still be respected in `PV.Spec.NodeAffinity` which is the desired API for volume topology as opposed to labels. So I think just wording the docs based on my comment above https://github.com/kubernetes/website/pull/24783#discussion_r514841131 would be sufficient.

Reply:
I'll adopt Andrew's words.  PTAL